Core/Core/Features/ActAsUser/ActAsUserWindow.swift

```diff
@@ -183,3 +199,32 @@ class ActAsUserOverlay: UIView {
         return nil
     }
 }
+
+// MARK: - Overlay Exceptions
+
+private extension UIViewController {
+
+    var isOverlayException: Bool {
```

Comment:
I would just call this method what it returns: if it's a system picker or not. The overlay word was also confusing for me because it's not a frequently used word for view controller presentation. Suggestion: `isSystemAssetPicker`.